README.md

```diff
@@ -1,11 +1,54 @@
-## My Project
+## ACK Multi-Controller Helm Chart
 
-TODO: Fill this README out!
+The ACK Multi-Controller Helm chart, shortened to `ack-chart`, acts as a wrapper
+for all ACK controllers released under General Availability (GA). 
 
-Be sure to:
+`ack-chart` lists each ACK controller as a subchart dependency.
 
-* Change the title in this README
-* Edit your repository description on GitHub
+`ack-chart` is currently in **developer preview** and is subject to breaking
+changes.
+
+## Usage
+
+Helm 3.8+ is required to install `ack-chart`, since it is hosted in an
+[OCI-based image registry][oci-registry].
+
+To install the latest version of `ack-chart`, use the following command:
+```bash
+export LATEST_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/ack-chart/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
+helm install --create-namespace -n ack-system ack-chart \
+  oci://public.ecr.aws/aws-controllers-k8s/ack-chart --version=$LATEST_VERSION
+```
+
+[oci-registry]: https://helm.sh/docs/topics/registries/
+
+### Enabling Subcharts
+
+By default, when installing the `ack-chart`, **none of the subcharts are
+installed**. To enable a subchart, set the value associated with the ACK
+controller. For example, to enable the `s3-controller`, enable the `s3-chart`
+like so:
+```yaml
+s3-chart:
+  enabled: true
+```
+
+Helm exposes all of the values for the underlying subcharts, just use the name
+of the subchart as the parent key and provide any overrides in the same
+structure as the subchart expects. For example, to update the `aws.region` for
+the `s3-chart`:
+```yaml
+s3-chart:
+  aws:
+    region: "us-east-1"
+```
+
+## Versioning
+
+`ack-chart` is versioned independently of the subcharts it contains. As any of
+the dependencies are updated, the version for `ack-chart` is also updated. If
+any of the dependencies is incremented by a minor version, the `ack-chart` is
+also incremented by a minor version.
```

Comment:
If we bump any of the controllers, it will bump this controller too. That way, subscribing to a patch version of the `ack-chart` will ensure you only get patch version increments of your current controllers. But if something bumps a minor version, we can't be sure that the consumer of `ack-chart` wants that upgrade.